### PR TITLE
Added linting using ESLint

### PR DESCRIPTION
**Pull request by harumijang**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,14 +3,24 @@
...
   "license": "MIT",
   "private": true,
-  "workspaces": ["starters/*", "packages/*"],
+  "workspaces": [
+    "starters/*",
+    "packages/*"
+  ],
   "scripts": {
-    "lint": "yarn --cwd starters/basic-starter next lint"
+    "lint": "yarn --cwd starters/basic-starter next lint",
```
Comment (lauriii) [inline package.json:11]: We should update this command so that it runs these newly added rules on CI

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
```
Comment (lauriii) [inline .eslintrc.js:1]: I think we can remove this file since we also have `eslintrc.json`

**Review by lauriii — CHANGES_REQUESTED** [review]

**Comment by harumijang** [comment]
I think the `lint` is getting executed now... Below image from running 
`./node_modules/.bin/eslint --cache --ext .ts,.tsx .`
image

 However, running `yarn lint` from the root looks like it also runs but then exits fails at the end. I am not sure what the fix should be.
```
next-acms-basic-starter:lint: ERROR: command finished with error: command (starters/basic-starter) yarn run lint exited (1)
command (starters/basic-starter) yarn run lint exited (1)
```

**Comment by lauriii** [comment]
Did you run `yarn install` in the monorepo before running the command? I get the same error if I try to run `yarn lint` without installing dependencies

**Comment by harumijang** [comment]
Failing on lint errors because there are many of them

**Comment by harumijang** [comment]
I'm not sure why it is still throwing a lint error even though I specified that `node` is a `DrupalNode`

image

**Pull request merged**